data_presentation.ipynb

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #53.        # Subplot 2: FFT Magnitude vs Frequency
</pre><p>This is not needed, just plot the time docmain</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1-24-HW5/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>